cmake/modules/FindCAP.cmake

```diff
@@ -0,0 +1,35 @@
+# - Try to find capabilities
+# Find the native CAP includes and library
+#
+#  CAP_FOUND - system has capabilities available
+#  CAP_INCLUDE_DIRS - where to find capabilities.h
+#
+#  Copyright (c) 2019 6WIND
+#
+#  Redistribution and use is allowed according to the terms of the New
+#  BSD license.
+#  For details see the accompanying COPYING-CMAKE-SCRIPTS file.
```

Comment:
This must be changed. At the very least the license has to be included in the repo.
But the "New BSD license" (aka 3-clause BSD license) contains additional provisions that can not be found in the MIT license. I don't know if that is a problem, but the easiest course of action would be to relicense under MIT.